tools/hooks/scary_changes.rb

```diff
@@ -66,12 +66,32 @@ def detect_special_files
     end
   end
 
+  def detect_dropbox_conflicts
+    changes = @all.grep(/'s conflicted copy/)
+    unless changes.empty?
+      puts red <<-EOS
+
+        Looks like you are adding dropbox conflicted copy files.
+        This is probably a mistake.
+
+#{changes.join("\n")}
+
+        If this change is intentional, you can bypass this message with the
+          --no-verify
+        flag.
+
+      EOS
+      raise "Commit blocked."
```

Comment:
nit: This could probably be refactored into a method `output_message_and_raise(message, changes)`, perhaps in another PR (happy to own).